packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -699,7 +689,8 @@ export default {
                 const data = {
                     basketId: this.basket.id,
                     customerNotes: {
-                        noteForRestaurant: this.userNote
+                        noteForRestaurant: this.userNotes.delivery || this.userNotes.restaurant,
+                        noteForKitchen: this.userNotes.kitchen || ''
```

Comment:
Aren't `noteForKitchen` and `noteForRestaurant` the same thing? Shouldn't it be something like `noteForRestaurant` and `noteForDriver`?

Reply:
I based this on slide decks I had seen before. These fields are for the call to order placement, so I'm not 100% on what their changes will be. 

But to add to the confusion, restaurant is the combined note field which we have now. Kitchen is the new one specifically for the kitchen (self explanatory). This part is something I will confirm with the order placement team.